### PR TITLE
EDU-529 Create new media player and use it within plugins

### PR DESCRIPTION
**Pull request by ahelmberger**
Closes https://educandu.atlassian.net/browse/EDU-529

### CODE REVIEW

File: src/components/media-player-controls.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Button, Slider } from 'antd';
+import { AudioMutedOutlined, CaretRightOutlined, PauseOutlined, SoundOutlined } from '@ant-design/icons';
+
+function formatMillisecondsAsDuration(milliseconds) {
```
Comment (adrianaluchian) [inline src/components/media-player-controls.js:7]: Now that we have the `media-utils` we could put this in there too and throw in a couple of tests.

File: src/plugins/anavis/anavis-display.js [hunk 2]
```diff
@@ -174,28 +59,31 @@ function AnavisDisplay({ content }) {
...
-        </div>)}
-
-      {media.text && (
-        <div className="Anavis-text">
-          <Markdown>{media.text}</Markdown>
-        </div>
-      )}
+      <div className={`AnavisDisplay-players u-width-${width || 100}`}>
+        {sourceUrl && (
+          <MediaPlayer
```
Comment (adrianaluchian) [inline src/plugins/anavis/anavis-display.js:75]: Nice! Did not realize the player was also used here.

**Comment by adrianaluchian** [comment]
Love the simplification in all related plugins! 😍

File: src/components/media-player-controls.js (hunk 1 above)
Comment (ahelmberger) [reply src/components/media-player-controls.js]: Good idea, thanks! It was put and tests were thrown :-)

**Review by adrianaluchian — APPROVED** [review]

**Pull request merged**